ini-templates/production-template.ini

```diff
@@ -5,6 +5,7 @@ accession_factory = ${accession_factory}
 blob_bucket = ${blob_bucket}
 blob_store_profile_name = ${file_upload_profile_name}
 file_upload_bucket = ${file_upload_bucket}
+accel_redirect_header = ${accel_redirect_header}
```

Comment:
Probably want this in base.ini so it applies to demos as well. @caseylitton can confirm.

Reply:
I thought demo instances ran production.ini? Removing the necessity to run nginx in development seems useful.

Reply:
Yeah, I think you are right. Concerned the `if proxy and accel_redirect_header` logic would change current behavior (local and demo) if the `accel_redirect_header` param wasn't available to both INIs.

Reply:
If that makes sense to you should we make symmetric change to snovault PR?